app/site/content/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Linear Algebra"
+---
+
+## Course Description
+This is a basic subject on matrix theory and linear algebra. Emphasis is given to topics that will be useful in other disciplines, including systems of equations, vector spaces, determinants, eigenvalues, similarity, and positive definite matrices.
```

Comment:
@pdpinch Just double checking, are we allowed to license this as BSD? Do you think it's short enough or do we already have permission for that?

Reply:
Can we use some lorem ipsum here?

Reply:
Can we use some lorem ipsum here?